### PR TITLE
Feature/core 438 general profile components

### PR DESCRIPTION
**Pull request by Tschuck**
- profile base setup
- profile verification cards
- migrated organizations into profile.vue

### CODE REVIEW

File: dapps/profile.vue/src/components/profile/profile.ts [hunk 1]
```diff
@@ -43,21 +45,34 @@ export default class ProfileDetailComponent extends mixins(EvanComponent) {
...
 
   /**
-   * profile information
+   * Current profile information
    */
-  accountId = '';
+  address = '';
   alias = '';
-  balance = '';
+  balance = 0;
```
Comment (karladler) [inline dapps/profile.vue/src/components/profile/profile.ts]: is currDate used for the balance? If yes we should it maybe put together with the balance into an object:

e.g.

```
balance: {
    amount,
    timestamp
}
```

File: dapps/profile.vue/src/components/profile/profile.ts [hunk 2]
```diff
@@ -43,21 +45,34 @@ export default class ProfileDetailComponent extends mixins(EvanComponent) {
...
     const runtime = (<any>this).getRuntime();
 
-    this.accountId = runtime.activeAccount;
-    this.alias = (await runtime.profile.getAddressBook()).profile[this.accountId].alias;
+    this.address = this.$route.params.address || runtime.activeAccount;
+    this.alias = (await runtime.profile.getAddressBook()).profile[this.address].alias;
+
+    // load balance and parse it to 3 decimal places
     this.balance = await dappBrowser.core.getBalance(runtime.activeAccount);
+    this.balance = Math.round(this.balance * 1000) / 1000;
```
Comment (karladler) [inline dapps/profile.vue/src/components/profile/profile.ts]: why not just `toFixed(3);` ?

File: dapps/profile.vue/src/components/profile/profile.ts [hunk 3]
```diff
@@ -43,21 +45,34 @@ export default class ProfileDetailComponent extends mixins(EvanComponent) {
...
+
+    // load verification status
+    // this.notaryIdentification = await getIdentificationDetails(runtime, this.address);
```
Comment (karladler) [inline dapps/profile.vue/src/components/profile/profile.ts]: usually uncommented code should not be checked in. Considering we will add it soon again we can do this maybe with a "TODO" flag.

File: dapps/profile.vue/src/components/profile/profile.vue [hunk 4]
```diff
@@ -25,42 +25,47 @@
...
-          {{ index + 1}}
-        </span>
-        <span>{{ step.title | translate }}</span>
-      </button>
-    </div>
+  <div class="profile-detail p-md-11 p-1">
+    <evan-loading v-if="loading"></evan-loading>
+    <template v-else>
+      <div class="row mb-3">
+        <div class="col-xl-4 col-lg-6 mb-3">
```
Comment (karladler) [inline dapps/profile.vue/src/components/profile/profile.vue]: looks like this is still the old layout. In the new one the wallet is in the right side

File: dapps/profile.vue/src/components/verifications/card/card.ts [hunk 5]
```diff
@@ -35,19 +35,20 @@ import { EvanComponent } from '@evan.network/ui-vue-core';
 import * as bcc from '@evan.network/api-blockchain-core';
 import * as dappBrowser from '@evan.network/ui-dapp-browser';
 
+/**
+ * Wrapper for profile verifications.
+ */
 @Component({ })
-export default class InfoDialogComponent extends mixins(EvanComponent) {
+export default class VerificationCardComponent extends mixins(EvanComponent) {
```
Comment (karladler) [inline dapps/profile.vue/src/components/verifications/card/card.ts]: maybe general card component?

File: dapps/profile.vue/src/components/verifications/notary/actions/pin/pin.vue [hunk 6]
```diff
@@ -32,91 +32,88 @@
...
-      <template v-slot:body id="pin-verification-modal">
-        <steps-indicator
+      <template v-slot:body>
+        <evan-steps
           :active-step="status"
           :steps="steps"
-          @updatestep="status = $event"
-        />
+          @updatestep="status = $event">
+        </evan-steps>
```
Comment (karladler) [inline dapps/profile.vue/src/components/verifications/notary/actions/pin/pin.vue]: why not self closing component?

File: dapps/profile.vue/src/components/verifications/notary/card/card.vue [hunk 7]
```diff
@@ -0,0 +1,177 @@
...
+          <evan-button type="link" size="sm"
+            class="d-block mt-1 text-muted"
+            @click="$refs.orgInfo.show();">
+            {{ '_profile.verifications.notary.learn-more' | translate }}
+          </evan-button>
+        </template>
+      </profile-verification-card>
+    </template>
+  </div>
+<!--     <evan-loading v-if="loading"></evan-loading>
```
Comment (karladler) [inline dapps/profile.vue/src/components/verifications/notary/card/card.vue]: https://kentcdodds.com/blog/please-dont-commit-commented-out-code :D

File: dapps/profile.vue/src/components/verifications/notary/info/info-content/info-content.vue [hunk 8]
```diff
@@ -0,0 +1,87 @@
+/*
+  Copyright (C) 2018-present evan GmbH.
+
+  This program is free software: you can redistribute it and/or modify it
+  under the terms of the GNU Affero General Public License, version 3,
+  as published by the Free Software Foundation.
+
```
Comment (karladler) [inline dapps/profile.vue/src/components/verifications/notary/info/info-content/info-content.vue:7]: above you removed the empty lines from the copyright. Do we have a common style for this?

File: dapps/profile.vue/src/components/profile/profile.ts (hunk 2 above)
Comment (Tschuck) [reply dapps/profile.vue/src/components/profile/profile.ts]: awkward :D

File: dapps/profile.vue/src/components/verifications/card/card.ts (hunk 5 above)
Comment (Tschuck) [reply dapps/profile.vue/src/components/verifications/card/card.ts]: - general verification card component exists in profile that uses this one
- eventually needs to be moved to vue-core?

File: dapps/profile.vue/src/components/verifications/notary/actions/pin/pin.vue (hunk 6 above)
Comment (Tschuck) [reply dapps/profile.vue/src/components/verifications/notary/actions/pin/pin.vue]: old habit from times without babel and ie7

File: dapps/profile.vue/src/components/verifications/notary/card/card.vue (hunk 7 above)
Comment (Tschuck) [reply dapps/profile.vue/src/components/verifications/notary/card/card.vue]: -.- :D

File: dapps/profile.vue/src/components/verifications/notary/info/info-content/info-content.vue (hunk 8 above)
Comment (Tschuck) [reply dapps/profile.vue/src/components/verifications/notary/info/info-content/info-content.vue:7]: This was seriously a mistake... Should include the line breaks.

File: dapps/profile.vue/src/components/verifications/card/card.ts (hunk 5 above)
Comment (karladler) [reply dapps/profile.vue/src/components/verifications/card/card.ts]: in vue core would be better I think, but as long as it's only used in profile it's also ok for me

**Review by karladler — APPROVED** [review]

**Pull request merged**